01-bash-scripting/03_ping_host_latency.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ $# -ne 1 ]; then
+    echo "Please provide exactly 1 argument!"
+    exit 1
+fi
+
+latency=$(curl --silent --output /dev/null -w "%{time_total}\n" "$1")
+
+echo "Latency for '$1' is $latency seconds"
```

Comment:
Just FYI: You could have put string output inside `-w` like following:
```
-w "Latency for '$1' is %{time_total} seconds"
```

And also should quote `latency` output as mentioned above.